timemachine/potentials/nonbonded.py

```diff
@@ -539,12 +539,12 @@ def coulomb_interaction_group_energy(q_ligand: Array, q_prefactors: Array) -> fl
 
     Parameters
     ----------
-    q_ligand: [N_lig] array
-    q_prefactors: [N_lig] array
+    q_ligand: [N_frames, N_lig] array
+    q_prefactors: [N_frames, N_lig] array
 
     Returns
     -------
-    energy: float
+    energy: [N_frames] array
     """
 
     return jnp.dot(q_prefactors, q_ligand)
```

Comment:
docstring was previously correct -- this function applies to a single frame / snapshot.

It can be mapped over multiple frames / snapshots (e.g. using a for loop or vmap or `process_traj_in_chunks`), but for clarity, it is implemented for a single snapshot.

Reply:
reverted